docs/reticulate.rst

```diff
@@ -0,0 +1,517 @@
+        synapseclient$Column(name = "Time", columnType = "DATE"))
+
+    schema <- synapseclient$Schema(name = "President Birthdays", columns = cols, parent = project)
+    table <- synapseclient$Table(schema, prez_birthdays)
+
+    # store the table in Synapse
+    table <- syn$store(table)
+
+We can query a table as in the following:
+
+  .. code-block::
+
+    tableId <- table$tableId
+
+    results <- syn$tableQuery(sprintf("select * from %s where Name='George Washington'", tableId))
+    results$asDataFrame()
+
+Wikis
+=====
+
+This example illustrates creating a wiki.
+
+See `here <index.html#wikis>`__ for more information on wiki APIs.
+
+  .. code-block::
+
+    content <- "
+    # My Wiki Page
+    Here is a description of my **fantastic** project!
+    "
+
+    # attachment
+    filePath <- tempfile()
+    connection <- file(filePath)
+    writeChar("this is the content of the file", connection, eos = NULL)
+    close(connection)
+    wiki <- synapseclient$Wiki(
+                owner = project,
+                title = "My Wiki Page",
+                markdown = content,
+                attachments = list(filePath)
+    )
+    wiki <- syn$store(wiki)
+
+An existing wiki can be updated as follows.
+
+  .. code-block::
+
+    wiki <- syn$getWiki(project)
+    wiki$markdown <- "
+    # My Wiki Page
+    Here is a description of my **fantastic** project! Let's
+    *emphasize* the important stuff.
+    "
+    wiki <- syn$store(wiki)
+
+
+Evaluations
+===========
+
+An Evaluation is a Synapse construct useful for building processing pipelines and
+for scoring predictive modeling and data analysis challenges.
+
+See `here <index.html#evaluations>`__ for more information on Evaluations.
+
+Creating an Evaluation:
+
+  .. code-block::
+
+    eval <- synapseclient$Evaluation(
+      name = sprintf("My unique evaluation created on %s", format(Sys.time(), "%a %b %d %H%M%OS4 %Y")),
+      description = "testing",
+      contentSource = project,
+      submissionReceiptMessage = "Thank you for your submission!",
+      submissionInstructionsMessage = "This evaluation only accepts files.")
+
+    eval <- syn$store(eval)
+
+    eval <- syn$getEvaluation(eval$id)
+
+Submitting a file to an existing Evaluation:
+
+  .. code-block::
+
+    # first create a file to submit
+    filePath <- tempfile()
+    connection <- file(filePath)
+    writeChar("this is my first submission", connection, eos = NULL)
+    close(connection)
+    file <- synapseclient$File(path = filePath, parent = project)
+    file <- syn$store(file)
+    # submit the created file
+    submission <- syn$submit(eval, file)
+
+List submissions:
+
+  .. code-block::
+
+    submissions <- syn$getSubmissionBundles(eval)
+
+    # submissions are returned as a generator
+    list(iterate(submissions))
+
+Retrieving submission by id:
+
+  .. code-block::
+
+    # Not evaluating this section because of SYNPY-235
```

Comment:
https://sagebionetworks.jira.com/browse/SYNPY-235 is closed.  Can we remove this line?